packages/scribe/tsconfig.json

```diff
@@ -1,12 +1,20 @@
 {
   "extends": "../../tsconfig.base",
   "compilerOptions": {
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": [
+        "src/*"
+      ]
+    }
   },
-  "include": ["src"],
+  "include": [
+    "src"
+  ],
   "references": [
     {
       "path": "./tsconfig.node.json"
     }
   ]
-}
+}
```

Comment:
BTW the red symbol on the line below says there isn't a blank line at the end of the file when there should be. Some old-school editors and command-line tools had weird issues without it so it is best to leave it in.